swarm/api/testapi.go

```diff
@@ -32,3 +36,30 @@ func NewControl(api *API, hive *network.Hive) *Control {
 func (c *Control) Hive() string {
 	return c.hive.String()
 }
+
+// DebugAPI is a umbrella structure to provide additional debug API endpoints
```

Comment:
well, Control had the same intention, plus the file is called `testapi`, need to sort out